### DISCUSSION TITLE
Expand doc for Nervous beginners

### DISCUSSION THREAD

**Discussion by skol101** [coqui-ai/TTS#959] [Qs for Live Q&A] [answered]
It wasn't clear for me beforehand that VITS VCTK model was trained on 44kHz dataset, even though original paper mentions that they have trained on the 22kHz dataset. 

That's why 'Resample' is set to true in the train_vits.py, because samples need to be resampled to the original dataset sample rate, if one is to continue training from existing model.

**Comment by erogol** [comment, marked as answer]
(answered in the call)
